lang/cs/Org.Apache.REEF.Client.Tests/YarnClientTests.cs

```diff
@@ -128,6 +132,139 @@ public void TestGetApplication()
+            var anyNewApplication = new NewApplication
+            {
+                ApplicationId = applicationId
+            };
+            restReqExecutor.ExecuteAsync<NewApplication>(
+                Arg.Is<IRestRequest>(
+                    req =>
+                        req.Resource == "ws/v1/cluster/apps/new-application"
+                        && req.Method == Method.POST),
+                anyUri,
+                CancellationToken.None).Returns(Task.FromResult(anyNewApplication));
+
+            var yarnClient = ctx.GetClient();
+            NewApplication actualNewApplication = yarnClient.CreateNewApplicationAsync().GetAwaiter().GetResult();
+
+            Assert.AreEqual(anyNewApplication, actualNewApplication);
+            urlProvider.Received(1).GetUrlAsync();
+        }
+
+        [TestMethod]
+        public void TestSubmitNewApplication()
+        {
+            var ctx = new TestContext();
+            var urlProvider = ctx.UrlProviderFake;
+            var restReqExecutor = ctx.RestRequestExecutorFake;
+            Uri anyUri = new Uri("anyscheme://anypath");
+            const string applicationId = "AnyApplicationId";
+            const string anyApplicationType = "REEFTest";
+            const string anyApplicationName = "AnyAPP";
+            urlProvider.GetUrlAsync().Returns(Task.FromResult(anyUri));
+            var anySubmitApplication = new SubmitApplication
+            {
+                ApplicationId = applicationId,
+                AmResource = new Resouce
+                {
+                    MemoryMB = 500,
+                    VCores = 1
+                },
+                ApplicationType = anyApplicationType,
+                ApplicationName = anyApplicationName,
+                KeepContainersAcrossApplicationAttempts = false,
+                MaxAppAttempts = 1,
+                Priority = 1,
+                UnmanagedAM = false,
+                AmContainerSpec = new AmContainerSpec
+                {
+                    Commands = new Commands
+                    {
+                        Command = @"DONTCARE"
+                    },
+                    LocalResources = new LocalResources
+                    {
+                        Entry = new List<KeyValuePair<string, LocalResourcesValue>>
+                        {
+                            new KeyValuePair<string, LocalResourcesValue>(
+                                "APPLICATIONWILLFAILBUTWEDONTCAREHERE",
+                                new LocalResourcesValue
+                                {
+                                    Resource = "Foo",
+                                    Type = ResourceType.FILE,
+                                    Visibility = Visibility.APPLICATION
+                                })
+                        }
+                    }
+                }
+            };
+
+            const string expectedJson = @"{""application-id"":""AnyApplicationId"",""application-name"":""AnyAPP""," +
```

Comment:
Would be great if you could format this to be more human readable, if it's not too much work.